ydb/public/api/protos/ydb_topic.proto

```diff
@@ -558,27 +561,39 @@ message StreamReadMessage {
         int64 direct_read_id = 2;
     }
 
+    // Signal from server that client has finished reading the partition and all messages has been read.
+    // Once a partition has been finished no further messages will ever arrive on that partition.
+    // Client should not send a response to the command.
+    message EndPartitionSession {
```

Comment:
How the message will work with `StopPartitionSessionRequest`?
Will I receive `StopPartitionSessionRequest` after `EndPartitionSession`?

Can server send `StopPartitionSessionRequest` with additional fields without add new message for notify about session finished?